tidb-cloud/release-notes.md

```diff
@@ -0,0 +1,36 @@
+---
+title: TiDB Cloud Release Notes in 2023
+summary: Learn about the release notes of TiDB Cloud in 2023.
+aliases: ['/tidbcloud/beta/supported-tidb-versions']
+---
+
+# TiDB Cloud Release Notes in 2023
+
+This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.
+
+## January 4, 2023
+
+**General changes**
+
+- Support scaling up TiDB, TiKV, and TiFlash nodes by increasing the **Node Size(vCPU + RAM)** for TiDB Dedicated Tier clusters hosted on AWS and created after December 31, 2022.
+
+    You can increase the node size [using the TiDB Cloud console](/tidb-cloud/scale-tidb-cluster.md#increase-node-size) or [using the TiDB Cloud API (beta)](https://docs.pingcap.com/tidbcloud/api/v1beta#tag/Cluster/operation/UpdateCluster).
+
+- Extend the metrics retention period on the [**Monitoring**](/tidb-cloud/built-in-monitoring.md) page to 2 days.
+
+    Now you have access to metrics data from the last 2 days, giving you more flexibility and visibility into your cluster performance and trends.
+
+    This feature comes at no additional cost and can be accessed on the **Diagnosis** tab of the [**Monitoring**](/tidb-cloud/built-in-monitoring.md) page for your cluster. This will help you identify and troubleshoot performance issues and monitor the overall health of your cluster more effectively.
```

Comment:
How to understand "additional cost"? What kind of cost it may be (money, time, machine)?  
```suggestion
    This improvement comes at no additional cost and can be accessed on the **Diagnosis** tab of the [**Monitoring**](/tidb-cloud/built-in-monitoring.md) page for your cluster. This will help you identify and troubleshoot performance issues and monitor the overall health of your cluster more effectively.
```

Reply:
it is is free?